.env.development

```diff
@@ -1,4 +1,4 @@
-REACT_APP_WEBMAP_DOMAIN=http://dev.treetracker.org
+REACT_APP_WEBMAP_DOMAIN=https://beta-map.treetracker.org
```

Comment:
It's the `.env.staging` and `.env.production` domains that should point at this URL.
@dadiorchen Is there a `dev` and `test` URL available for the beta web map?